detox/ios/Detox/Invocation/Element.swift

```diff
@@ -171,6 +171,12 @@ class Element : NSObject {
 		view.dtx_replaceText(text)
 	}
 	
+	func accessibilityAction(_ actionName: String) {
+		let action = view.accessibilityCustomActions!.first(where: { $0.name == actionName })
```

Comment:
Please add checks that the action actually exists before attempting to perform it. If not, fail the test with an explanation why.

Reply:
Fixed this in my commits